joomla/joomla-cms
# [4.0] Global configuration options add media-breakpoint-down(sm)

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/27613#issuecomment-577838109 .

### Summary of Changes
I added a `media-breakpoint-down(sm)` for the options. On small devices the label ob an option is above this option.


### Testing Instructions

1.   Open Global Configuration an see that all options displayed beside the label.
2.   Apply this patch and run `npm ci`
3.   Open Global Configuration again and see, that the options on small devices is displayed in there own row above the label.


### Expected result
![Global Configuration   test   Administration(2)](https://user-images.githubusercontent.com/9974686/73020624-4d03de80-3e26-11ea-8536-74af28fe4bd4.png)



### Actual result
![Global Configuration   test   Administration(3)](https://user-images.githubusercontent.com/9974686/73020631-52f9bf80-3e26-11ea-8d73-18abd691012f.png)



### Documentation Changes Required
No
